src/app.ts

```diff
@@ -2,19 +2,88 @@
 
 import Bugsnag from '@bugsnag/react-native'
 import { asObject, asString } from 'cleaners'
-import { Text, TextInput } from 'react-native'
+import { LogBox, Text, TextInput } from 'react-native'
 import RNFS from 'react-native-fs'
 
+import ENV from '../env.json'
 import { changeTheme, getTheme } from './components/services/ThemeContext'
-import { ENV } from './env'
 import { NumberMap } from './types/types'
 import { log, logToServer } from './util/logger'
 
+console.log = () => {}
+console.info = () => {}
+console.warn = () => {}
+
+const realConsDbg = console.debug
+
+type FgColor = 'b' | 'r' | 'g' | 'c' | 'm' | 'y'
+type BgColor = 'B' | 'R' | 'G' | 'C' | 'M' | 'Y'
+
+console.debug = (message?: any, color?: FgColor | BgColor | [FgColor, BgColor] | string) => {
```

Comment:
I've never been a fan of overloading console methods. This certainly is a lot of mucking about with our debug method. These log utils should be reliable and without unexpected behavior, and so all of this seems risky for the win of getting some colors? I'm not in favor of this. Make this overloading an option for developers if they want to add colors to their console.debug method and either with an ENV flag or a new utility like `prettyDebug` or something that they can import from `app.ts` while debugging. This also seems more suited for a scratch file.

Reply:
Sorry this entire file wasn't supposed to end up in this review